cmd/serverless/main.go

```diff
@@ -162,6 +162,9 @@ func runAgent(stopCh chan struct{}) (serverlessDaemon *daemon.Daemon, err error)
 		}
 	}
 
+	// output Datadog-prefixed environment variables if log level is set to DEBUG for support purpose
```

Comment:
I think you can get rid of this comment, maybe rename the function so it's more clear what it's for instead.

Reply:
Another option would be to add the comment above the function definition as a doc string. That way you can see the explanation by hovering over the function, but it doesn't need to be explained at the call site.